src/main/java/org/spongepowered/common/registry/SpongeGameRegistry.java

```diff
@@ -397,7 +394,7 @@
     private final Map<String, CookedFish> cookedFishMappings = Maps.newHashMap();
     private final Map<String, DyeColor> dyeColorMappings = Maps.newHashMap();
     private final Map<String, Art> artMappings = Maps.newHashMap();
-    private final Map<String, EntityType> entityTypeMappings = Maps.newHashMap();
+    protected final Map<String, EntityType> entityTypeMappings = Maps.newHashMap();
```

Comment:
I believe @Mumfrey is going to clean up `SpongeGameRegistry` at some point so this will hopefully be improved when that happens.